tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -145,8 +200,75 @@ def test_positive_validate_katello_certs_check_output(self, cert_setup, cert_dat
             )
         self.validate_output(result, cert_data)
 
+    @pytest.mark.tier1
+    def test_katello_certs_check_output_wildcard_inputs(self, generate_certs, cert_data):
+        """Validate that katello-certs-check generates correct output with wildcard certs.
+
+        :id: 7f9da806-5b23-11eb-b7ea-d46d6dd3b5b2
+
+        :steps:
+
+            1. Get valid wildcard certs from generate_certs
+            2. Run katello-certs-check with the required valid arguments
+               katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
+               -b CA_BUNDLE_FILE
+            3. Assert the output has correct commands with options
+
+        :expectedresults: katello-certs-check should generate correct commands
+         with options.
+        """
+        with get_connection() as connection:
+            result = connection.run(
+                'katello-certs-check -c {} -k {} -b {}'.format(
+                    "certs/wildcard.crt",
+                    "certs/wildcard.key",
+                    "certs/ca.crt",
+                ),
+                output_format='plain',
+            )
+        self.validate_output(result, cert_data)
+
+    @pytest.mark.parametrize("error, cert_file, key_file, ca_file", invalid_inputs)
+    @pytest.mark.tier1
+    def test_katello_certs_check_output_invalid_input(
+        self, generate_certs, error, cert_file, key_file, ca_file
+    ):
+        """Validate that katello-certs-check raise the correct errors for invalid
+         inputs
+
+        :id: 37742f5e-598a-11eb-a349-d46d6dd3b5b2
+
+        :steps:
+
+            1. Get invalid certs from generate_certs
+            2. Run katello-certs-check with the required valid arguments
+               katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
+               -b CA_BUNDLE_FILE
+            3. Assert the output has correct error with message
+
+        :expectedresults: Katello-certs-check should raise error when it receives the invalid
+         inputs.
+        """
+        with get_connection() as connection:
+            result = connection.run(
+                'katello-certs-check -c {} -k {} -b {}'.format(
```

Comment:
Please use f-string

Reply:
I would have done f-strings it would make code little more ugly and non-understable and based I made it based on the coding standard https://github.com/SatelliteQE/robottelo/blame/master/docs/code_standards.rst#L47. Even you think this can be written little more nicely let me know I can change it.

Reply:
There is certainly room for personal preference here, if you'd like to keep it how it is.

I think its plenty readable even with the dict references, not a blocking opinion though :)

Consider using multi-line strings when writing commands like this.

```
result = connection.run(
    'katello-certs-check '
    f'-c {dict_var["first_key"]} '
    f'-k {dict_var["second_key"]} '
    f'-b {dict_var["third_key"]}'
)
```